docs/hardware/devices/heltec-sensor/index.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: heltec-sensors
+title: Heltec® Plug&Play Sensors
+sidebar_label: HELTEC® Sensors
+sidebar_position: 7
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<Tabs
+groupId="heltec"
+queryString="heltec"
+defaultValue="v3.0"
+values={[
+{label: 'Capsule V3.0', value: 'v3.0'}
+]}>
+<TabItem value="v3.0">
+
+## Heltec Capsule Sensor Rev. 3.0
+
+- **MCU:**
+  - ESP32-S3FN8 (WiFi & Bluetooth)
+- **LoRa Transceiver:**
+  - Semtech SX1262
+- **Frequency Options:**
+  - 433 MHz
+  - 470 - 510 MHz
+  - 863 - 870 MHz
+  - 902 - 928 MHz
+- **Connectors:**
+  - Magnetic suction interface
+  - Antenna:
+    - Dedicated 2.4 GHz SMT antenna for WiFi/Bluetooth
+    - Dedicated SMT antenna for LoRa
+
+### How to upload firmware
+
+:::info
+Capsule Sensor V3 uses WirelessBoot mode to upload firmware, exchange information, and print logs through WiFi.
+That is, whether you update the firmware locally or via the Web. You need to get the device into WirelssBoot state first. See this link for instructions on how to use it: 
+:::
+- [**Wireless Boot**](https://docs.heltec.org/en/node/esp32/capsule_sensor_v3/wireless_boot.html#).
+### Touch button/Physical button differences
+
+:::warning
+Because the touch button version is easy to accidentally action while close to metal or put it in the pocket, Heltec has abandoned the production of this version, but still a small number of samples have entered the market.
```

Comment:
Perhaps: "Because the touch button is easy to accidentally activate while close to metal or in your pocket, Heltec has discontinued production of this version.  However, a small number of samples have entered the market."